build.gradle.kts

```diff
@@ -380,11 +404,29 @@ publishing {
             artifact(sourcesJar.get()) {
                 classifier = "sources"
             }
+            // TODO: we probably will have to configure and add javadoc, as it seemingly required by Maven Central
+            // artifact(tasks.javadoc)
 
             pom {
                 this.name.set(name)
                 this.description.set("Lincheck - framework for testing concurrent data structures")
 
+                url.set("https://github.com/JetBrains/lincheck")
+                scm {
+                    connection.set("scm:git:https://github.com/JetBrains/lincheck.git")
+                    url.set("https://github.com/JetBrains/lincheck")
+                }
+
+                developers {
+                    developer {
+                        this.name.set("Nikita Koval")
+                        id.set("ndkoval")
```

Comment:
If you want to configure the publication with my account, the correct id is `nikita.koval`